src/js/post-911-gib-status/containers/Post911GIBStatusApp.jsx

```diff
@@ -56,7 +57,8 @@ class Post911GIBStatusApp extends React.Component {
                   print a copy of this screen for benefit and eligibility verification.
                 </p>
               </div>
-              <UserInfoSection userData={enrollmentState}/>
+              <UserInfoSection userData={enrollmentData}/>
+              <EnrollmentHistory enrollmentData={enrollmentData}/>
```

Comment:
Just noticed the props we pass to these two sections are called different things (`userData` and `enrollmentData`). Should we make it consistent since the value is the same? I don't care which one it is.